coffee/base.py

```diff
@@ -675,6 +684,15 @@ def size(self):
         return self.sym.rank or (0,)
 
     @property
```

Comment:
Why is it a property? You could get the same thing by simply reading and writing `self.core`.